### ISSUE TITLE
Focus styles fall below min contrast ratio (WCAG Level AA)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2246]
The yellow should not be used alone to focus or highlight elements, because it doesn't have enough contrast against white.

Affects:

- [x] Tables
- [x] Tabs
- [x] Highlighting
- [x] Borderless checkboxes and radios
- [x] Ghost buttons
- [x] Password checkbox

Page 66-67 of the June 2022 DAC DS report...

---

## Colour Contrast [Non-Text]
Some of the colour combinations found on the service are low contrast and are likely to be difficult for people with low vision to distinguish.

WCAG Reference:
1.4.11 Non-text Contrast (Level AA)
Understanding Non-text Contrast |How to Meet Non-text Contrast

### Issue ID: DAC_Colour_Contrast_Non_Text_01

URL: https://ons-design-system-prototypes.netlify.app/prototypes/dac-may2022/table-1.html#tabId2
Page title: Tabs and sortable and scrollable tables
Journey: Journey 5, Step 13

### Screenshot:
![Screenshot 2022-06-16 at 15 09 13](https://user-images.githubusercontent.com/43346934/174088758-e757217f-73ba-4147-b941-f3ed3822aab3.png)

Tables can be navigated onto by keyboard-only users to allow keyboard-only users to scroll the content. The table once receiving focus shows a yellow outline, and the yellow outline could be difficult for the user to see as it is below the minimum contrast for non-text contrast. The contrast failure also applies to the tabs that are used to switch between tables.

Foreground: #FBC900
Background: #FFFFFF
The contrast ratio is: **1.6:1**
1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

### Current code ref(s):

#tabId1 
```
<section id="tabId1" class="ons-tabs__panel" tabindex="0" role="tabpanel" aria-labelledby="tab_tabId1"> <table class="ons-table ons-table--sortable" data-aria-sort="Sort by" data-aria-asc="ascending" data-aria-desc="descending"> <caption class="ons-table__caption">Sortable table example</caption> […] </table>
```

### Solution:
GOV services counteract the yellow outline contrast failure with a black border. The light-yellow contrasts well against dark colours, and the black contrasts well against light colours.
We advise visiting GOV Design System Colour and following the colour palette located on the page. Specifically, focus state and border.
